fdbclient/FDBTypes.h

```diff
@@ -908,7 +908,7 @@ struct TLogVersion {
 		MIN_SUPPORTED = V2,
 		MAX_SUPPORTED = V7,
 		MIN_RECRUITABLE = V6,
```

Comment:
Does this also need updated to `V7`?

Reply:
We talked about this one and the `MIN_RECRUITABLE` should be updated to `V7`.

Reply:
@sfc-gh-yiwu 7.1 branch has been cut, can you update `MIN_RECRUITABLE` so that this can be merged?